### DISCUSSION TITLE
ComboBox direction of opening

### DISCUSSION THREAD

**Discussion by ray-5050** [emilk/egui#1532] [Q&A]
I want to put a combo box in the bottom panel but unless the space below the combo is as large as the list, 3 in this case, the list is not fully displayed.  I would prefer not to make the bottom panel larger since it would be just empty space.  Can the combo box be opened upwards?  I didn't see any way in the code but I may have missed it. Thanks for your help?

**Comment by emilk** [comment]
https://github.com/emilk/egui/issues/693
